starter-code/src/main.js

```diff
@@ -40,11 +42,58 @@ $(document).ready(function(){
   });
 
   // Add all the div's to the HTML
-  document.getElementById('memory_board').innerHTML = html;
+  $('#memory_board').html(html);
   // Bind the click event of each element to a function
-$('.back').on('click', function () {
-   
-});
-});
+  var cardsCounter = 0;
+  var card1 = 0;
+  var card2 = 0;
+  var cardComparison = false;
+  var clickedDisplay = $("#pairs_clicked");
+  var guessedDisplay = $("#pairs_guessed");
+
+  $('.back').on('click', function () {
+    if (cardsCounter === 0) {
+      cardsCounter += 1;
+      card1 = $(this);
+      flipBack(card1);
+    } else if (cardsCounter === 1) {
+      card2 = $(this);
+      flipBack(card2);
 
+      //Would be better to check if the card.attr("name") is already in the array
+      memoryGame.pickedCards.push(card1.attr("name"));
+      memoryGame.pickedCards.push(card2.attr("name"));
+      console.log(memoryGame.pickedCards);
+      console.log(memoryGame.pairsClicked+1);
+      console.log(memoryGame.pairsGuessed+1);
+      cardComparison = memoryGame.checkIfPair(card1,card2);
+      if (cardComparison === true) {
```

Comment:
you could simply write `if (cardComparison) `, or even better, name the variable "isPair" and then write `if (isPair)`